src/EditorFeatures/Test2/IntelliSense/OldCompletion/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -0,0 +1,3914 @@
+            End Using
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function EnumCompletionNotTriggeredOnPlusCommitCharacter() As Task
+            Await EnumCompletionNotTriggeredOn("+"c)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function EnumCompletionNotTriggeredOnLeftBraceCommitCharacter() As Task
+            Await EnumCompletionNotTriggeredOn("{"c)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function EnumCompletionNotTriggeredOnSpaceCommitCharacter() As Task
+            Await EnumCompletionNotTriggeredOn(" "c)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function EnumCompletionNotTriggeredOnSemicolonCommitCharacter() As Task
+            Await EnumCompletionNotTriggeredOn(";"c)
+        End Function
+
+        Private Async Function EnumCompletionNotTriggeredOn(c As Char) As Task
+            Using state = OldCompletionTestState.CreateCSharpTestState(
+                <Document>
+enum Numeros { Uno, Dos }
+class Goo
+{
+    void Bar()
+    {
+        Numeros num = $$
+    }
+}
+                </Document>)
+
+                state.SendTypeChars("Nu")
+                Await state.WaitForAsynchronousOperationsAsync()
+                Await state.AssertSelectedCompletionItem(displayText:="Numeros", isHardSelected:=True)
+                state.SendTypeChars(c.ToString())
+                Await state.WaitForAsynchronousOperationsAsync()
+                Assert.NotEqual("Numberos", state.CurrentCompletionPresenterSession?.SelectedItem?.DisplayText)
+                Assert.Contains(String.Format("Numeros num = Nu{0}", c), state.GetLineTextFromCaretPosition(), StringComparison.Ordinal)
+            End Using
+        End Function
+
+        <WorkItem(544296, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/544296")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestVerbatimNamedIdentifierFiltering() As Task
+            Using state = OldCompletionTestState.CreateCSharpTestState(
+                              <Document>
+class Program
+{
+    void Goo(int @int)
+    {
+        Goo($$
+    }
+}
+                              </Document>)
+
+                state.SendTypeChars("i")
+                Await state.AssertCompletionSession()
+                Assert.True(state.CurrentCompletionPresenterSession.CompletionItems.Any(Function(i) i.DisplayText = "@int:"))
+                state.SendTypeChars("n")
+                Await state.WaitForAsynchronousOperationsAsync()
+                Assert.True(state.CurrentCompletionPresenterSession.CompletionItems.Any(Function(i) i.DisplayText = "@int:"))
+                state.SendTy
```

Comment:
❗️ Failure to copy these files directly to the subfolder resulted in a failure to preserve the file's encoding. This is the wrong `i`.